src/frontend/src/layouts/NavTopBar/NavUserMenu.tsx

```diff
@@ -115,7 +115,7 @@ const NavUserMenu: React.FC = () => {
           <ListItemText>Settings</ListItemText>
         </MenuItem>
         {auth.user?.role === 'ADMIN' || auth.user?.role === 'APP_ADMIN' ? <AdminTools /> : null}
-        {process.env.NODE_ENV === 'development' ? <DevLogout /> : <ProdLogout />}
```

Comment:
can u go back to the development one because otherwise local logout doesnt work for me for some reason